### PR TITLE
Plans and pricing update

### LINKED ISSUE #3730: Amendment to AI/ML Add-on table
The $40,000 one off fee includes both the workshop and the Assessment, so the two cards needs to be made into one. See update in the plans and pricing page. 

An includes also used on the ai-index page so please update this as well. 

AI index page: https://docs.google.com/document/d/1mJq8PxH8FVRVF9i1Xzh0f6M73eyjqZE4NaKYHRcRAKM/edit#heading=h.9tf8ath3tck5

Plans and pricing page: https://docs.google.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit#heading=h.sh3ke0r5zpcp

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Update pricing on include

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/ai> and <http://0.0.0.0:8001/support/plans-and-pricing>
- See the 'AI add-on for Kubernetes Discoverer and Discoverer Plus' section matches the corresponding section in the [copy doc](https://docs.google.com/a/canonical.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit?disco=AAAACDy-B1I)

## Issue / Card

Fixes #3730

### CODE REVIEW

**Comment by steverydz** [comment]
@anasereijo I think this section will need some design direction :)

**Comment by anasereijo** [comment]
Hey @steverydz can we please make something like this?

File: templates/shared/pricing/_kubernetes-consulting-add-ons.html [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
-      <li class="p-list__item is-ticked">Full pipeline view</li>
-      <li class="p-list__item is-ticked">ML / Data science assessment</li>
-    </ul>
-    <p><a href="/kubernetes/contact-us?product=kubernetes-discoverer">Contact us&nbsp;&rsaquo;</a></p>
-  </div>
-  <div class="col-6 p-card">
-    <div class="p-card__header">
-      <h2 class="p-heading--four">ML / Data Science Assessment</h2>
-      <p><span class="p-heading--two">$40,000</span> one off fee</p>
+    <div class="row u-equal-height p-divided">
```
Comment (Caleb-Ellis) [inline templates/shared/pricing/_kubernetes-consulting-add-ons.html]: Should be `p-divider` (not that it'll actually make a difference with `u-equal-height` already there)

Comment (steverydz) [reply templates/shared/pricing/_kubernetes-consulting-add-ons.html]: Fixed

**Comment by anasereijo** [comment]
@steverydz please remove the vertical line between the 2 lists and then it's ready :)

**Comment by steverydz** [comment]
@anasereijo Done

**Pull request merged**